### DISCUSSION TITLE
I want to know what this means

### DISCUSSION THREAD

**Discussion by Lapius7** [jagrosh/MusicBot#1692] [Questions]
What is the difference between 'linear' and 'fair' 'QueueType'?

**Comment by MichailiK** [comment]
Linear & Fair queues behave differently when a user adds a track to the queue.

The linear queue simply puts the added track to the end of the queue.

Fair queue may add the track closer to the beginning of the queue depending on who requested it. You can observe fair queue's behavior by adding lots of tracks as one user, then adding 1 track as a different user.
